src/item/lookup.js

```diff
@@ -24,20 +24,20 @@ define([
 		// 1: Due to #16, never use the cached resolved non-leaf nodes. It may not
 		//    represent its leafs in its entirety.
 		value = resourceGet( Cldr._resolved, normalizedPath );
-		if ( value && typeof value !== "object" /* 1 */ ) {
+		if ( typeof value === "string" /* 1 */ ) {
 			return value;
 		}
 
 		// Check raw data
 		value = resourceGet( Cldr._raw, normalizedPath );
 
-		if ( !value ) {
+		if ( typeof value === "undefined" ) {
```

Comment:
I did so to follow the same style as in line 16 of this file. I will simplify it.